extensions/script-libraries/qr/qr.livecodescript

```diff
@@ -1522,7 +1559,13 @@ command qrCreate pImg, pData, pECC, pSize, pMask
       # calculate minimum version required
       local tMinVersion
       put 0 into tMinVersion
-      repeat with tVersion = 1 to 40
+      
+      if pMinVersion is empty then put 1 into pMinVersion
```

Comment:
@livecodepanos I realise it's not your code but is there any chance you could change all these to multi-line `if`? The single line form rubs me up the wrong way.